bin/crew

```diff
@@ -1268,6 +1268,20 @@ def shrink_dir(dir)
   end
 end
 
+def install_files(src, dst = src.delete_prefix('.'))
```

Comment:
How about?
```
def install_files(src, dst = File.join(CREW_PREFIX, src.delete_prefix('./usr/local')))
```
That way we can pass any destination directory from the binary and it automagically installs in the correct location.

Reply:
This will not work for `install_files(CREW_DEST_HOME, HOME)`

Reply:
Right, because source and destination are reversed?

Reply:
Right because it needs to be `install_files(File.join('.', CREW_DEST_HOME), HOME)`?

Reply:
The `HOME` parameter should override the `dst` default.